i18n-scripts/set-english-defaults.js

```diff
@@ -48,11 +49,14 @@ function updateFile(fileName) {
       switch (determineRule(keys[i])) {
         case 0:
           [originalKey] = keys[i].split('_other');
-          updatedFile[keys[i]] = `{{count}} ${pluralize(originalKey)}`;
+
+          wordWithNoCount = originalKey.replace('{{count}}', '').trim()
```

Comment:
Are these changes in this file intentional? Feels like those don't belong to this PR...

Reply:
No i pushed in the wrong branch

Reply:
fixing